logfire/_internal/cli.py

```diff
@@ -77,6 +77,11 @@ def parse_whoami(args: argparse.Namespace) -> None:
 
 def parse_clean(args: argparse.Namespace) -> None:
     """Remove the contents of the Logfire data directory."""
+    if Path.cwd() == Path.home():
+        sys.stderr.write('The clean command cannot be run in the home directory. ')
+        sys.stderr.write('Please run the command from a different directory.\n')
```

Comment:
This doesn't make sense to me. How should the user clean out the files if they happen to be in the home dir? Can't we just delete the specific files?